src/core/SIPTransactions/UASInviteTransaction.cs

```diff
@@ -121,12 +121,12 @@ public new void SendFinalResponse(SIPResponse sipResponse)
         /// generated final response.
         /// </summary>
         /// <returns>A socket error with the result of the cancel.</returns>
-        public void CancelCall()
+        public void CancelCall(SIPRequest sipCancelRequest)

```

Comment:
This breaks the public API surface and will affect library users. Can you make it optional?

Reply:
Sure thing, I have now made it optional. Please let me know if you like me to change SIPRequest to be nullable like "public void CancelCall(SIPRequest? sipCancelRequest = null)".